warehouse/routes.py

```diff
@@ -539,6 +539,19 @@ def includeme(config):
         domain=warehouse,
     )
 
+    config.add_route(
+        "api.echo",
+        "/api/echo",
```

Comment:
thought: do we want to use `/api-unstable/` or something as a prefix?

its one more way for us to make clear that this is a no-commitment zone.

Reply:
`/danger-api/` is the prefix for now, and we'll swap it back when we're ready for v1.